docs/src/app/components/raw-code/floating-action-button-code.txt

```diff
@@ -6,7 +6,7 @@
 
 <FloatingActionButton iconClassName="muidocs-icon-action-grade" secondary={true}>
```

Comment:
@shaurya947 sorry, I thought you were talking about my change on line 8/9.  Line 7 is before this, and yes, it should have a closing tag, but that's not what I was changing.